### PR TITLE
Update Chinese translation of "chrome_devtools" and "bidirectional"

### PR DESCRIPTION
**Pull request by alaahong**
**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Types of changes

- [ ] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [ ] Code example added (and I also added the example to all translated languages)
- [X] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [X] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [X] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

File: website_and_docs/content/documentation/webdriver/bidirectional/_index.zh-cn.md [hunk 1]
```diff
@@ -1,26 +1,31 @@
...
-with the ability to stream events from the user agent to the controlling software via WebSockets,
-better matching the evented nature of the browser DOM.
 
-Because it's a bad idea to tie your tests to a specific version of a specific browser,
-the Selenium project recommends using WebDriver BiDi wherever possible.
-However, until the spec is complete there are many useful things that the CDP offers.
-To help keep your tests independent and portable, Selenium offers some useful helper classes.
-At the moment, these use the CDP, but when we shall be using WebDriver Bidi as soon as possible
+严格限制请求响应命令的传统webdriver模型, 
+将从user agent转变为基于WebScokets的软件控制,
```
Comment (harsha509) [inline website_and_docs/content/documentation/webdriver/bidirectional/_index.zh-cn.md]: WebSockets

File: website_and_docs/content/documentation/webdriver/bidirectional/_index.zh-cn.md [hunk 2]
```diff
@@ -1,26 +1,31 @@
...
-The traditional webdriver model of strict request/response commands will be supplemented
-with the ability to stream events from the user agent to the controlling software via WebSockets,
-better matching the evented nature of the browser DOM.
 
-Because it's a bad idea to tie your tests to a specific version of a specific browser,
-the Selenium project recommends using WebDriver BiDi wherever possible.
-However, until the spec is complete there are many useful things that the CDP offers.
-To help keep your tests independent and portable, Selenium offers some useful helper classes.
-At the moment, these use the CDP, but when we shall be using WebDriver Bidi as soon as possible
+严格限制请求响应命令的传统webdriver模型,
```
Comment (harsha509) [inline website_and_docs/content/documentation/webdriver/bidirectional/_index.zh-cn.md]: it may be a mistake, but we always write it as WebDriver AFAIK!

**Review by harsha509 — CHANGES_REQUESTED** [review]
HI @alaahong,

Thanks for the PR.

Could you please look into the review comments!

File: website_and_docs/content/documentation/webdriver/bidirectional/_index.zh-cn.md (hunk 2 above)
Comment (alaahong) [reply website_and_docs/content/documentation/webdriver/bidirectional/_index.zh-cn.md]: Got, just replaced all places with WebDriver in bidirectional

File: website_and_docs/content/documentation/webdriver/bidirectional/_index.zh-cn.md (hunk 1 above)
Comment (alaahong) [reply website_and_docs/content/documentation/webdriver/bidirectional/_index.zh-cn.md]: My mistake, just updated.

**Review by harsha509 — APPROVED** [review]
Thank you @alaahong!

**Pull request merged**